From 8b1be2cb71d3bc1fb8b419de22b10d937bf3774b Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Thu, 28 May 2020 23:05:16 +0200
Subject: [PATCH] [CMake] typo

But we should stop provide pkg-config compatibility, as it is not
tested, but still able to break stuff.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0cf4d7e..ab3b952 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -73,7 +73,7 @@ ENDIF(EIGEN_NO_AUTOMATIC_RESIZING)
 ADD_PROJECT_DEPENDENCY(pinocchio 2.3.1 REQUIRED
   PKG_CONFIG_REQUIRES "pinocchio >= 2.3.1")
 ADD_PROJECT_DEPENDENCY(eiquadprog 1.1.3 REQUIRED
-  PKG_CONFIG_REQUIRES "eiquaprog >= 1.1.3")
+  PKG_CONFIG_REQUIRES "eiquadprog >= 1.1.3")
 
 SET(BOOST_COMPONENTS filesystem system unit_test_framework)
 
-- 
2.26.2

